addons/website/views/snippets/s_references.xml

```diff
@@ -2,27 +2,28 @@
 <odoo>
```

Comment:
Commit message nitpick: 
The limit is 72 chars, which you respected, but you have to use these 72 chars => you can move "default" at the previous line :wink: